src/scanner/scanNFTStats.ts

```diff
@@ -0,0 +1,404 @@
+import {BlockHash, EventRecord} from "@polkadot/types/interfaces";
+import { trackCancelSaleData } from "./utils/trackSaleCancel";
+import {
+	trackAuctionData,
+	processAuctionSoldEvent,
+	trackAuctionBundleData,
+} from "./utils/trackTokenAuction";
+import { trackBurnBatchData, trackBurnData } from "./utils/trackTokenBurn";
+import { trackBuyData } from "./utils/trackTokenBuy";
+import {
+	trackAdditionalTokenData,
+	trackTokenSeriesData,
+	trackUniqueMintData,
+} from "./utils/trackTokenCreation";
+import { trackSeriesNameData } from "./utils/trackTokenName";
+import { trackSellBundleData, trackSellData } from "./utils/trackTokenSell";
+import {
+	trackTransferBatchData,
+	trackTransferData,
+} from "./utils/trackTokenTransfers";
+import { trackBidData } from "./utils/trackBidData";
+
+import { Api } from "@cennznet/api";
+import { config } from "dotenv";
+import { logger }  from "../logger";
+import { Vec } from "@polkadot/types-codec";
+import { u8aToString } from "@polkadot/util";
+import mongoose from 'mongoose';
+import {SignedBlock} from "@polkadot/types/interfaces/runtime";
+import {fetchNFTBlockForRange} from "./utils/fetchNFTBlockNumberForRange";
+import {updateProcessedBlockInDB} from "./dbOperations";
+const { LastBlockScan } = require('../mongo/models');
+config();
+export let supportedAssets = [];
+
+// find the event for the extrinsic
+function filterExtrinsicEvents(extrinsicIdx, events) {
```

Comment:
all of these helper functions should be put at the bottom for better brevity or chuck them in the `commonUtils.ts` like you have